artacle-swagger.yaml

```diff
@@ -825,6 +827,641 @@ paths:
+      description: Returns particular token valuation
+      parameters:
+        - name: cid
+          in: path
+          required: true
+          description: Artacle collection ID as returned by the /collections/ endpoint
+          schema:
+            type: integer
+            example: 78
+        - name: tid
+          in: path
+          required: true
+          description: Token ID as specified in a smart contract (for example, 1 for Bored Apes Yacht Club or 78000001 for Art Blocks Fidenza)
+          schema:
+            type: integer
+            example: 78000001
+        - name: apiKey
+          in: query
+          required: true
+          description: API access key
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - data
+                properties:
+                  data:
+                    type: object
+                    properties:
+                      tokenId:
+                        type: string
+                        description: Token ID as per smart contract (for example, 1 for Bored Apes Yacht Club or 78000001 for Art Blocks Fidenza)
+                        example: 78000001
+                      tokenName:
+                        type: string
+                        description: Token name
+                        example: Floris Veilstorm
+                      AValuation:
+                        type: number
+                        description: Token A-Valuation
+                        format: float
+                        example: 0.0414
+                      AValuationUSD:
+                        type: number
+                        description: Token A-Valuation in USD
+                        format: float
+                        example: 102.768876
+                      topTraitValuation:
+                        type: number
+                        description: Token trait valuation
+                        format: float
+                        example: 0.0414
+                      topTraitValuationUSD:
+                        type: number
+                        description: Token trait valuation in USD
+                        format: float
+                        example: 0.0414
+                      rarityReward:
+                        type: number
+                        description: Token rarity reward
+                        format: float
+                        example: 0.0414
+                      rarityRewardUSD:
+                        type: number
+                        description: Token rarity reward in USD
+                        format: float
+                        example: 0.0414
+                      isSpecialCase:
```

Comment:
Actually now I'm thinking of 'isSpecialCaseValuation' - the same flag is used in /tokens/v1, and there it loses its context. Adding 'Valuation' in the name will help to keep it related to valuation.